src/probot.ts

```diff
@@ -93,15 +93,7 @@ export class Probot {
 
     this.webhooks = getWebhooks(this.state);
 
-    this.on = (eventNameOrNames, callback) => {
```

Comment:
This one is a breaking change, just pointing it out so you know to expect it

Reply:
is this change necessary because of the new types? Anyway to avoid it? It will be quite impactful. I want to make this change eventually anyway, but if we do it now we at least need to add a deprecation to the current version

Reply:
Yes, the change is required because of the new types because support for the `*' event was removed in `@octokit/webhooks` v8

It would require rewriting a bunch of the types from `@octokit/webhooks` just to make this change backwards compatible
I don't think there is any way to avoid it